client/package.json

```diff
@@ -24,6 +24,7 @@
     "@angular/platform-browser": "~11.2.1",
     "@angular/platform-browser-dynamic": "~11.2.1",
     "@angular/router": "~11.2.1",
+    "component": "^1.1.0",
```

Comment:
I am very sure you didn't want to install this package. You should run `npm remove component`.